internal/accountstable/accountstable.go

```diff
@@ -14,6 +15,86 @@ import (
 	"github.com/SIGBlockchain/project_aurum/internal/sqlstatements"
 )
 
+// structs
+
+// ATConnection struct used for
+type ATConnection struct {
```

Comment:
Can we rename this to `AccountsTableConnection`?

Reply:
Originally I was going to do this but the go-linter said this:

[click this link](https://ibb.co/CJGq515)

Thoughts?

Reply:
Interesting. I guess we can make it that.

In your comments can you replace `at` with `accounts table`, because right now it looks like the word "at" and it's a bit confusing.